ClassicsKotlin/app/src/main/java/com/android/tv/classics/utils/TvLauncherUtils.kt

```diff
@@ -37,7 +36,6 @@ import com.android.tv.classics.models.TvMediaCollection
 
 /** Collection of static methods used to handle Android TV Home Screen Launcher operations */
 @RequiresApi(26)
-@SuppressLint("RestrictedApi")
```

Comment:
Unfortunately the referenced library is restricted, so this lint suppression is needed here.

Reply:
This removal can be merged now that [bug #138150076](https://issuetracker.google.com/138150076) has been resolved.